interfaces/cython/cantera/data/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path 
+
+def files(ext=''):
+	here = Path(__file__).parent
+	data_files = [x.name for x in here.iterdir() if not x.name.startswith('__') and x.name.endswith(ext)]
+	return data_files
```

Comment:
Please make sure the file ends with one blank line. You can set your editor to do both of these things automatically, depending on which editor you're using the instructions vary.